### DISCUSSION TITLE
Dock button has disappeared from lyrics window

### DISCUSSION THREAD

**Discussion by Logansfury** [audacious-media-player/audacious#1433] [Q&A] [answered]
I was shown a picture of audacious 4.4 running with the lyrics plugin window docked in the left side of the player. I tried to replicate but every time I clicked the dock button the display went on top of the player instead of to the left side as in the example picture. I could find no way to move the lyrics from top to left, and the last time that I disengaged it from being docked, the dock button just disappeared entirely from the window. All it has now is minimize, fullscreen and the less-than-fullscreen enlargement.

How do I get the dock button back please?

**Comment by radioactiveman** [comment]
I assume you used the Qt interface initially and later changed to the GTK interface. They behave a bit different regarding docking windows.
Whether you use Qt or GTK is shown in the Audacious settings tab "Appearance".

Qt: Forget about the "dock button" as you call it. Simply drag the plugin window at the position where you want to dock it.

![Screenshot](https://github.com/audacious-media-player/audacious/assets/1301412/2329431b-b6dd-4d58-b14a-1a6ac93c1f29)

GTK: Right-click below the title bar where a label displays the plugin name. This video explains it in more detail: https://www.youtube.com/watch?v=Zfe5WAFmgZg

**Comment by Logansfury** [comment]
Thank you for the explanation and the link, I will watch the video.

You have excellent taste in music, I am listening to Pink Floyd right now!
(Animals album)

**Comment by Logansfury** [comment]
I am afraid I am not seeing what is being show in the video:

[image: aud.png]

I tried both right clicking and left clicking in both the Lyrics title bar,
and in the window with the lyric text. Nothing I do is showing Dock Left,
Dock Right etc

What should I do?

**Comment by radioactiveman** [comment]
Then you probably use Qt again. The mentioned video is for GTK.
Read my post from above again and check what is shown in the "Appearance" settings tab.

Edit: And your screenshot aud.png isn't showing up here. This would help too. :)

**Reply by Logansfury** [reply]
Hello, thank you for the latest reply. I am sorry but I am indeed on GTK. Here is a link to a screenshot of my preferences. As you will see, I am in GTK interface, and the Lyrics window's dock button is just gone:

Screengrab on imgur: https://i.imgur.com/6ByF6kN.png

**Reply by radioactiveman** [reply, marked as answer]
> I tried both right clicking and left clicking in both the Lyrics title bar,
and in the window with the lyric text. Nothing I do is showing Dock Left,
Dock Right etc

You should not click on the title bar, but further down, in the red area.

> GTK: Right-click below the title bar where a label displays the plugin name. This video explains it in more detail: https://www.youtube.com/watch?v=Zfe5WAFmgZg

![grafik](https://github.com/user-attachments/assets/4707e26e-ba2c-48c9-9ffc-3c2698c3ccb6)


And to also answer your initial question:
Switch to the Qt mode (Audacious Settings -> Appearance) again and you will get the dock button back.

**Comment by Logansfury** [comment]
Hello Thomas, how are you?

the grafik.png just gave me everything I needed without having to change
modes, I now know the correct place to click and my dock options are
appearing as expected.

Thank you so much for taking the time to explain the features to me :)
